binding/unity/Assets/Rhino/RhinoManager.cs

```diff
@@ -130,6 +130,19 @@ public void Process()
             VoiceProcessor.Instance.StartRecording(_rhino.FrameLength, _rhino.SampleRate);
         }
 
+        /// <summary>
+        /// Resets the internal state of Picovoice. It should be called before processing a new stream of audio 
+        /// or when process was stopped while processing a stream of audio.
+        /// </summary>
+        public void Reset()
+        {
+            if (_rhino == null)
+            {
+                throw new RhinoInvalidStateException("Cannot reset RhinoManager - resources have already been released");
+            }
+            _rhino.Reset();
+        }
```

Comment:
because we don't expose stop in Rhino manager, we don't need to expose reset. `Process()` ensures that we reach a result.